containers/dotnet-postgres/README.md

```diff
@@ -0,0 +1,153 @@
+# C# (.NET) and MS SQL
+
+## Summary
+
+*Develop C# and .NET Core based applications. Includes all needed SDKs, extensions, dependencies and a PostgreSQL container for parallel database development. Adds an additional PostgreSQL container to the C# (.NET Core) container definition.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team, [Berkays](https://github.com/Berkays)  |
```

Comment:
I'm assuming you are okay with helping to maintain this definition?  If so, you can add yourself as the contributor.
```suggestion
| *Contributors* | [Berkays](https://github.com/Berkays)  |
```